# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gnome-print10/glibc.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur gnome-print-0.35-orig/libgnomeprint/parseTT.c gnome-print-0.35/libgnomeprint/parseTT.c
--- gnome-print-0.35-orig/libgnomeprint/parseTT.c	2009-08-23 20:02:24.000000000 +0000
+++ gnome-print-0.35/libgnomeprint/parseTT.c	2009-08-23 20:04:04.000000000 +0000
@@ -362,7 +362,7 @@
 static gint subfrombase (gint from, gint to);
 static int gssentry (struct gentry *ge, struct stem *hs, gshort *hpairs, gint nhs, struct stem *vs, gshort *vpairs, gint nvs, struct stembounds *s, gshort *egp, gint *nextvsi, gint *nexthsi);
 static int findstemat (gint value, gint origin, struct gentry *ge, struct stem *sp, gshort *pairs, gint ns, gint prevbest);
-static void getline (gchar **src);
+static void getLine (gchar **src);
 static gchar *eexec_start (gchar *line);
 static gchar *eexec_string (gchar *string);
 static const gchar *eexec_byte (guchar c);
@@ -1593,7 +1593,7 @@
 
 	while (*(src + cnt) != 0) {
 		string = src + cnt;
-		getline (&string);
+		getLine (&string);
 		cnt = string - src;
 		if (!strcmp (line, "currentfile eexec\n")) {
 			if ((buf = eexec_start (line)) != NULL) {
@@ -5506,7 +5506,7 @@
 
 /* ttf_type1_dump */
 static void
-getline (gchar **src)
+getLine (gchar **src)
 {
 	gint c;
 	gchar *p = line;
